File: c/02-estrutura-condicional/dardo.c
/*Problema "dardo"
No arremesso de dardo, o atleta tem três chances para lançar o dardo à maior distância que conseguir.
Você deve criar um programa para, dadas as medidas das três tentativas de lançamento, informar qual
foi a maior.*/

#include <stdio.h> 
int main () {
    double d1, d2, d3;

    printf("Digite as tres distancias: \n");
    scanf("%lf %lf %lf", &d1, &d2, &d3);

    if (d1 >= d2 && d1 >= d3) {
        printf("MAIOR DISTANCIA: %.1lf\n", d1);
    } else if (d2 >= d1 && d2 >= d3) {
        printf("MAIOR DISTANCIA: %.1lf\n", d2);
    } else {
        printf("MAIOR DISTANCIA: %.1lf\n", d3);
    }

    return 0;
}

File: java/02-estrutura-condicional/Dardo.java
/*Problema "dardo"
No arremesso de dardo, o atleta tem três chances para lançar o dardo à maior distância que conseguir.
Você deve criar um programa para, dadas as medidas das três tentativas de lançamento, informar qual
foi a maior.*/

import java.util.Locale;
import java.util.Scanner;

public class Dardo {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double a, b, c;
		
		System.out.println("Digite as tres distancias: ");
		a = sc.nextDouble();
		b = sc.nextDouble();
		c = sc.nextDouble();
		
		if (a >= b && a >= c) {
			System.out.println("MAIOR DISTANCIA = " + String.format("%.2f", a));
		} else if (b >= a && b >= c) {
			System.out.println("MAIOR DISTANCIA = " + String.format("%.2f", b));
		} else {
			System.out.println("MAIOR DISTANCIA = " + String.format("%.2f", c));
		}
		
		sc.close();
	}

}
